### ISSUE TITLE
timedelta can't set explicit time

### ISSUE THREAD

**Issue by kungfoome** [joke2k/faker#607]
https://github.com/joke2k/faker/blob/1cbb8b0dd9854254c65dab86dc95b7cc8b112a74/faker/providers/date_time/__init__.py#L373

A few things from this.
1. I would have expected it to be in seconds and not days by default.
2. I can't override the behavior to set an explicit delta. Meaning, I can't parse something like '434323s' if I wanted to.

I would at least propose to switch it to seconds by default since this deals with timestamps which are in seconds anyway.

**Comment by fcurella** [comment]
@kungfu71186 https://github.com/joke2k/faker/blob/1cbb8b0dd9854254c65dab86dc95b7cc8b112a74/faker/providers/date_time/__init__.py#L370

this line should be able to parse a string `'434323s'`. Isn't it working?

**Comment by kungfu71186** [comment]
Let me try again. I thought I had.
